scripts/deploy_broker.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+#
+# Helper script for deploying the Broker to an existing OpenShift cluster
+# using an OpenShift template.
+#
+
+#
+# VERIFY and UPDATE the variables below
+#
+CLUSTER_ADMIN_USER="system:admin"
+TEMPLATE_URL=${TEMPLATE_URL:-"https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml"}
+DOCKERHUB_ORG=${DOCKERHUB_ORG:-"ansibleplaybookbundle"} # DocherHub org where APBs can be found, default 'ansibleplaybookbundle'
```

Comment:
TYPO in the comment: DocherHub should be DockerHub

Reply:
thanks, I'll update